schema.graphql

```diff
@@ -23,3 +23,47 @@ type AccountMultisig @entity {
   multisig: Account!
   signatory: Account!
 }
+
+enum EventStatus {
+  approve
+  reject
+}
+
+type MultisigEvent @entity {
+  id: ID!
+  address: String!
+  status: EventStatus!
+  blockCreated: Int!
+  indexCreated: Int!
+  multisig: MultisigOperation!
+  timestamp: BigInt!
```

Comment:
why big int?

Reply:
take it from subquery-history project

Reply:
let's make it Int, it's default timestamp format